packages/stems/src/components/HarmonyButton/HarmonyButton.module.css

```diff
@@ -212,6 +212,17 @@
   --text-color: var(--background-white);
 }
 
+/* Plain */
+.plain {
+  --button-color: transparent;
+  --text-color: var(--neutral);
+  background: transparent;
+  border: none;
+  border-radius: 0;
+  padding: 0;
+  height: 100%;
```

Comment:
do we need height?

Reply:
Yeah at least I didn't find another way to prevent this. I think because the other buttons want that vertical space to push the border out? Couldn't figure out what was creating it though.
<img width="346" alt="image" src="https://github.com/AudiusProject/audius-client/assets/2358254/e8a4d132-06a2-4ccf-a57d-d1f1a61acf92">